src/convertToObject.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const keyValuePairs = sourceString.split(';')
+    .map(line => line.trim())
+    .filter(key => key.length > 1)
+    .map(solution => solution.split(':'));
+
+  const result = {};
+
+  keyValuePairs.map(([key, value]) => {
```

Comment:
`map` is usually used to create new array, in this case, it would be better to change it to forEach or try to use reduce